api/desecapi/serializers.py

```diff
@@ -1,6 +1,71 @@
 from rest_framework import serializers
-from desecapi.models import Domain, Donation, User
+from desecapi.models import Domain, Donation, User, RRset
 from djoser import serializers as djoserSerializers
+import json
+
+
+class JSONSerializer(serializers.Field):
+    def to_representation(self, obj):
+        return json.loads(obj)
+
+    def to_internal_value(self, data):
+        return json.dumps(data)
+
+
+class RecordsSerializer(JSONSerializer):
+    def to_internal_value(self, records):
+        if not all(isinstance(record, str) for record in records):
+            msg = 'Incorrect type. Expected a list of strings'
```

Comment:
In a later commit, this was changed to check `isinstance(records, str) or not all(isinstance(record, str) for record in records)` because inputting a string would pass the check in the way it was done previously. I believe this is safe now -- can you let me know your thoughts?